packages/prisma/zod-utils.ts

```diff
@@ -304,6 +305,7 @@ export const schemaBookingCancelParams = z.object({
   allRemainingBookings: z.boolean().optional(),
   cancellationReason: z.string().optional(),
   seatReferenceUid: z.string().optional(),
+  cancelledBy: z.string().nullish(),
```

Comment:
```suggestion
  cancelledBy: z.string().email({ message: "Invalid email" }).optional(),
```
Also, why was this nullish but rescheduledBy optional? 🤔

Reply:
Yeah my bad this should be optional too